clash-lib/src/Clash/Primitives/DSL.hs

```diff
@@ -456,6 +465,7 @@ boolFromBit boolName = \case
         [ (Just (BitLit H), Literal Nothing (BoolLit True))
         , (Nothing        , Literal Nothing (BoolLit False))
         ]
+    declareUseOnce (Proc Blocking) uniqueBoolName
```

Comment:
I don't think we ever build clocked processes with this DSL, so I would expect `Proc NonBlocking` here. Does it fail if you do?

Reply:
I think you're right, I looked at https://github.com/clash-lang/clash-compiler/blob/df3d5d1484413f2366e64804dab760fdb57edf14/clash-lib/src/Clash/Netlist/BlackBox.hs#L226 and I think `Proc NonBlocking` will do exactly what we want across VHDL and Verilog.